sbom/report_collection.go

```diff
@@ -39,13 +39,17 @@ type BomAsset struct {
 }
 
 type BomPackage struct {
-	Name     string   `json:"name,omitempty"`
-	Version  string   `json:"version,omitempty"`
-	Purl     string   `json:"purl,omitempty"`
-	CPEs     []string `json:"cpes.map,omitempty"`
-	FilePath string   `json:"file.path,omitempty"`
-	Format   string   `json:"format,omitempty"`
+	Name    string   `json:"name,omitempty"`
+	Version string   `json:"version,omitempty"`
+	Purl    string   `json:"purl,omitempty"`
+	CPEs    []string `json:"cpes.map,omitempty"`
+	// used by python packages
+	FilePath string `json:"file.path,omitempty"`
```

Comment:
Couldn't we use `FilePaths` with only one element?

This reads like it will cause confusion in the future.

Reply:
This is required for parsing the results from the python resource. This resource needs to be adjusted first to return multiple entries. The plan is to move the python package to return multiple files as well. Once that is done, we can remove the single entry here. We cannot deprecate it here until the resource is updated. The proto already deprecated the single location field exactly for that reason.

Reply:
Will adjust once the python resource is updated.